frontend/frontend-react/src/components/DeleteMultipleRacksForm.js

```diff
@@ -61,28 +65,28 @@ export class DeleteMultipleRacksForm extends Component {
     let start_rack;
     let end_rack;
     return (
-      <div>
-        <Button variant="outlined" onClick={() => this.props.sendShowTable(true)}>Back</Button>{' '}
+      <Container maxwidth="s">
+        <Link to='/racks/'><Button variant="outlined">Back</Button>{' '}</Link>
         <form onSubmit={this.handleSubmit}>
           <Grid container spacing={1}>
             <Grid item xs={12}>
-              <h1>Create Multiple Racks</h1>
+              <h1>Delete Racks</h1>
             </Grid>
-            <Grid item xs={6}>
+            <Grid item xs={3}>
               <TextField label='Deletion Range Start' type="text" fullWidth
                          onChange={e => this.setState({rack_num_start: e.target.value})}/>
             </Grid>
-            <Grid item xs={6}>
+            <Grid item xs={3}>
               <TextField label='Deletion Range End' type="text" fullWidth
                          onChange={e => this.setState({rack_num_end: e.target.value})}/>
             </Grid>
             <Grid item xs={12}>
-              <Button variant="contained" type="submit" color="primary" onClick={() => this.handleSubmit}>Create
-                +</Button>{' '}
+              <Button variant="contained" type="submit" color="primary" onClick={() => this.handleSubmit}>Delete
+                -</Button>{' '}
```

Comment:
'optional' messaging should be consistent between creation and deletion. How do I delete a specific single rack?

Reply:
ah ig it's from the list view i click a trash can

Reply:
u think thats ok or should we also have the option to delete in a single rack there too? or is that more confusing because you can delete 1 rack from 2 places

Reply:
We could make the button on the rack tab initally say 'delete rack range?'